### DISCUSSION TITLE
CRUD icons in add form don't show correctly

### DISCUSSION THREAD

**Discussion by ndikaros** [fabiocaccamo/django-admin-interface#350] [Q&A]
Happy 2024 every one! I would like to ask for some suggestions.
In the add form of a model, the CRUD icons appear to be stacked in paragraphs instead of being in the same row. Like in the following image
![image](https://github.com/fabiocaccamo/django-admin-interface/assets/20741798/d251c5e9-2899-4189-b1be-31472260f22e)
Contrary,  when  i view the page in small screen mode they seem to be perfectly fine as follows.
![image](https://github.com/fabiocaccamo/django-admin-interface/assets/20741798/3c4ad984-b9a7-4039-8b5c-a3d0a2b89b4a)
Has anyone an idea on what could possibly cause this? Maybe a django version  (i use the version 4.1.8)?
Thanks in advance for your help.

**Comment by fabiocaccamo** [comment]
@ndikaros this has been fixed in the latest version (`0.28.2`).

**Comment by ndikaros** [comment]
Yes it worked fine! i was using 0.28.1 for some reason. Thanks a lot!

**Reply by fabiocaccamo** [reply]
👍
